docs/dev/cli/sauce-connect-proxy.md

```diff
@@ -167,57 +191,45 @@ HTTP Header Injection is disabled for all HTTPS domains passed to the `--no-ssl-
 
 ### `--tunnel-domains`
 
-<p><small>| OPTIONAL | STRING | <span className="sauceRed">4.6.x</span> <span className="sauceRed">4.7.x</span> <span className="sauceRed">4.8.x</span>| </small></p>
+<p><small>| OPTIONAL | STRING | <span className="sauceGreen">4.7.x</span> <span className="sauceGreen">4.8.x</span>| </small></p>
 
 **Description**: Sets domain(s) that are requested through the Sauce Connect Proxy tunnel. This is the inverse of [`--direct-domains`](#--direct-domains). When adding multiple domains, [format them as a comma-separated list](#formatting-domains). See [Tuning Sauce Connect Proxy Traffic](/secure-connections/sauce-connect/proxy-tunnels/#tunnel-domains) for more information.<br/>
 **Environment variable**: `SAUCE_TUNNEL_DOMAINS`<br/>
 **Shorthand**: `-t`
 
 ---
 
-### `--rest-url`
-
-<p><small>| OPTIONAL | STRING | <span className="sauceRed">4.6.x</span> <span className="sauceRed">4.7.x</span> <span className="sauceRed">4.8.x</span>| </small></p>
-
-**Description**: Sets the URL for the [data center endpoint](/basics/data-center-endpoints) of the location where the device you're testing on is hosted.<br/>
-**Environment variable**: `SAUCE_REST_URL`<br/>
-**Shorthand**: `-x`
-
-```bash
-#US-West-1
-./sc -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -x https://api.us-west-1.saucelabs.com/rest/v1
+### `--no-remove-colliding-tunnels`
 
-#EU-Central-1
-./sc -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -x https://api.eu-central-1.saucelabs.com/rest/v1
+<p><small>| OPTIONAL | </small></p>
 
-#APAC-Southeast-1
-./sc -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -x https://api.apac-southeast-1.saucelabs.com/rest/v1
-```
+**Description**: Prevents the removal of colliding tunnels (that is, tunnels with the same name). <br/>
+**Environment variable**: n/a <br/>
+**Shorthand**: n/a
 
 :::note
-Effective with version 4.7.0, this flag was deprecated and replaced by [`--region`](/dev/cli/sauce-connect-proxy/#--region).
+Effective with version 4.7.0, this flag was deprecated and replaced by [`--tunnel-pool`](#--tunnel-pool).
 :::
 
 ---
 
-### `--no-remove-colliding-tunnels`
+### `--extra-info`
```

Comment:
The section for `--extra-info `here should be removed:
<img width="870" alt="Screenshot 2023-05-05 at 7 12 10 PM" src="https://user-images.githubusercontent.com/15352964/236583278-e530dd73-8f4f-4569-914f-37fe3124c51d.png">